lib/graphql/schema/field.rb

```diff
@@ -271,6 +271,7 @@ def initialize(type: nil, name: nil, owner: nil, null: nil, description: :not_gi
         else
           true
         end
+        @skip_nodes_on_raise = skip_nodes_on_raise
```

Comment:
This is just a hack to make this value avaiable in `#type`.

Should this be encoded in the `@return_type_expr` somehow?